tests/semantics/manifest.jsonld

```diff
@@ -317,13 +317,13 @@
       ]
     },
     {
-      "@id": "trs:constrained-bnodes-in-embedded-fail",
+      "@id": "trs:constrained-bnodes-in-quoted-fail",
       "@type": "NegativeEntailmentTest",
-      "comment": "Embedded bnode identifiers share the same scope as non-embedded ones. A bnode occuring both in embedded triples and in asserted triples must statisfy them all at the same time.",
+      "comment": "Embedded bnode identifiers share the same scope as non-quoted ones. A bnode occuring both in quoted triples and in asserted triples must statisfy them all at the same time.",
```

Comment:
```suggestion
      "comment": "Quoted bnode identifiers share the same scope as non-quoted ones. A bnode occurring both in quoted triples and in asserted triples must satisfy all occurrences at the same time.",
```

Reply:
Updated in 1d4775ca95660f1dbe2b15848b04e93c32ddc5c4.